src/sections/Footer/index.js

```diff
@@ -1,170 +1,219 @@
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Events
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Mailing List
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Internships
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    GSOC 2020
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    GSOD 2020
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Community Bridge
+                  </Link>
+                </li>
+              </ul>
+            </div>
+          </Col>
+          <Col xs={6} sm={4} lg={3}>
+            <div className="footer-widgets">
+              <h3 className="widget-title">Company</h3>
+              <ul className="widget-catagory">
+                <li>
+                  <Link className="anchor" to="#">
+                    News
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Brand
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Careers
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Partners
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Contact
+                  </Link>
+                </li>
+              </ul>
+            </div>
+          </Col>
+          </Row>
+          <Row className="footer-contact">
+            <Col className="footer-widgets" xs={12} sm={6} lg={7}>
+              <Language />
+            </Col>
+            <Col className="footer-widgets" xs={12} sm={6} lg={5}>
+              <h3 class="widget-title">Contact</h3>
+              <Row className="contactf">
+                <Link className="social-icons" to="#">
+                  <img height="40 px" src={mail_icon} />
+                </Link>
+                <Link className="social-icons" to="#">
+                  <img height="40 px" src={slack_icon} />
+                </Link>
+                <Link className="social-icons" to="#">
+                  <img height="40 px" src={twitter_icon} />
+                </Link>
+                <Link className="social-icons" to="#">
+                  <img height="40 px" src={github_icon} />
+                </Link>
+                <Link className="social-icons" to="#">

```

Comment:
Given that each of these links are to external locations, we can go ahead an include the actual links now for each of the buttons.